src/index.ts

```diff
@@ -331,6 +332,8 @@ function main() {
                         answers.dockerTag = 'testing';
                     } else if (program.dockerTagOverride) {
                         answers.dockerTag = program.dockerTagOverride;
+                    } else if (program.setLocalEnvironments) {
```

Comment:
I don't think this should be in else if. Should be after line 325. If it is not set then it should be undefined or null